app/controllers/summary_controller.rb

```diff
@@ -0,0 +1,33 @@
+class SummaryController < ApplicationController
+
+  def index
+    @presenter = SummaryPresenter.new(results, search_params)
+  end
+
+private
+
+  def results
+    Page.find_by_sql(["select pages.id, pages.base_path, count(page_visits.visit_id) as total_pageviews, count(distinct(page_visits.visit_id)) as unique_visits from pages join page_visits on page_visits.page_id = pages.id where pages.base_path like ? group by pages.id, pages.base_path order by #{search_params[:sort_key]} #{search_params[:sort_direction]};", "%#{search_params[:q]}%"])
```

Comment:
If this is only supposed to return pages which have survey answers, as the UI says, it doesn't seem right.  eg, if you search for "credit" and order by unique pageviews ascending, the top result is https://govuk-user-intent-survey-explorer.cloudapps.digital/pages/5153, which doesn't have any respondents.

Reply:
Interesting. From what I understand if we have a page in the data then it should be default have a response, as the data is made up entirely of survey responses and pages that users have been to as part of their journey (i.e. won't don't include all pages on GOV.UK in the data if they haven't been visited as part of a survey response). I'll investigate this issue with the data but I suspect that it is a separate issue, with the data.